setup.py

```diff
@@ -12,31 +12,21 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-import inspect
 import os
 import setuptools
-import sys
 
 requirements = [
     "requests>=2.19",
     "setuptools>=40.1.0",
     "qiskit-terra>=0.16.0",
 ]
 
-
-if not hasattr(setuptools,
-               'find_namespace_packages') or not inspect.ismethod(
-                    setuptools.find_namespace_packages):
-    print("Your setuptools version:'{}' does not support PEP 420 "
-          "(find_namespace_packages). Upgrade it to version >='40.1.0' and "
-          "repeat install.".format(setuptools.__version__))
-    sys.exit(1)
+PACKAGES = setuptools.find_namespace_packages(exclude=['test*'])
```

Comment:
You don't need find_namespace_package here, that's only for namespace pacakages (like it was before). You can use find_packages instead